apps/web/src/components/Basenames/RegistrationBackground/index.tsx

```diff
@@ -16,14 +20,16 @@ export default function RegistrationBackground() {
   const isPending = registrationStep === RegistrationSteps.Pending;
   const isSuccess = registrationStep === RegistrationSteps.Success;
 
-  const claimBackgroundClasses = classNames(
-    'pointer-events-none absolute inset-0 w-full h-full bg-cover bg-center -z-10',
```

Comment:
double checking you intend to remove the negative from z-10 here

Reply:
yeah it's on these classes

grayVideoBackgroundClasses
blueVideoBackgroundClasses